ecl/hql/hqlgram2.cpp

```diff
@@ -9766,9 +9766,22 @@ void HqlGram::checkDerivedCompatible(IIdAtom * name, IHqlExpression * scope, IHq
     }
 }
 
+bool HqlGram::checkSignedMacros()
```

Comment:
style: (picky).  The prefix "check" implies that a function is going to look at something and then perform an action based on the result.  Better to have a name that fits well inside a boolean context e.g. if ([is]insideSignedMacro())